ownCloud/Client/Actions/MoreViewHeader.swift

```diff
@@ -100,19 +100,13 @@ class MoreViewHeader: UIView {
 			labelContainerView.bottomAnchor.constraint(lessThanOrEqualTo: self.safeAreaLayoutGuide.bottomAnchor, constant: -20)
 		])
 
-		titleLabel.attributedText = NSAttributedString(string: item.name, attributes: [NSAttributedString.Key.font: UIFont.systemFont(ofSize: 17, weight: .semibold)])
+		titleLabel.attributedText = NSAttributedString(string: item.name ?? "", attributes: [NSAttributedString.Key.font: UIFont.systemFont(ofSize: 17, weight: .semibold)])
```

Comment:
@felix-schwarz minor complain.. magic number 17 could be replaced with constant. Would make sense if this font size is used in other places..

Reply:
I totally agree. The Theme-system in the app should also provide styles like this, but this is outside the scope of the file provider. I see an opportunity to standardize these styles alongside making other `Theme` changes in `feature/branded-login` (#168).